src/main/style/checkstyle.xml

```diff
@@ -25,6 +25,10 @@
     <module name="TreeWalker">
         <module name="AvoidStarImport" />
         <module name="Indentation">
+            <!-- Indentation: 4 spaces per indentation level (aligns with formatter) -->
```

Comment:
This configuration doesn't change anything, because 4 is already the default value. See https://checkstyle.sourceforge.io/checks/misc/indentation.html#Properties

Reply:
Consistency is already ensured by the default value. It's good practice not to overwrite a default value with the same value.